app/test_main.py

```diff
@@ -1,3 +1,38 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("arg1, arg2", [(15, 15), ])
+def test_type_return_get_human_age(arg1, arg2):
+    assert isinstance(get_human_age(arg1, arg2), list)
+
+
+@pytest.mark.parametrize("arg1, arg2", [(15, 15), ])
+def test_len_list_return_get_human_age(arg1, arg2):
+    assert len(get_human_age(arg1, arg2)) == 2
+
+
+@pytest.mark.parametrize("args", [(15, 15)])
+def test_type_return_value_in_list_get_human_age(args):
+    f = True
+    for i in range(len(get_human_age(*args))):
+        f = isinstance(get_human_age(*args)[i], int)
+    assert f
```

Comment:
```suggestion
    assert = all(isinstance(i, int) for i in get_human_age(*args))
```
if you need check output types maybe simplify a little ;)